packages/datadog-instrumentations/src/helpers/hooks.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   '@opensearch-project/opensearch': () => require('../opensearch'),
   '@opentelemetry/sdk-trace-node': () => require('../otel-sdk-trace'),
   '@redis/client': () => require('../redis'),
+  '@smithy/smithy-client': () => require('../aws-sdk'),
```

Comment:
Do we want to instrumentate `@smithy/smithy-client` when `aws-sdk` is not loaded?